Sources/WorkflowUIKit/UIKitPresenter.swift

```diff
@@ -86,6 +86,8 @@ open class UIKitPresenter: OrchestrationResponder {
             }
         } else if let presenting = first.presentingViewController {
             presenting.dismiss(animated: animated, completion: onFinish)
```

Comment:
I could be misremembering how dismiss works, but doesn't it dismiss the view you call it on?  But this seems to be dismissing up to the view we are calling it on.  That makes me wonder if there are some exploratory scenarios we should look at (not necessarily unit test) around chaining workflows in different scenarios.

Or you know, if y'all already did all of that, then we're good and you can tell me to stop.

Reply:
We should check to see if having `ViewController` present another viewcontroller (I'll call this `ViewControllerB`) which in turn launches a workflow. Does the dismiss send us back to `ViewControllerB` or back to `ViewController` and which is the appropriate/expected behavior?

Reply:
Sending back to `ViewControllerB` is what I think the correct behavior should be.  I've abandoned my Workflow so all things related to my workflow I closed, but not the thing that launched the workflow.

Reply:
`dismiss` has separate functionalities depending on the calling view controller.  We are forcing it to always use the behavior where the calling view controller relies on the presenting view controller to dismiss.